src/Tests/dotnet.Tests/dotnet-msbuild/GivenDotnetOsArchOptions.cs

```diff
@@ -143,36 +143,5 @@ public void CommandsRunWithArchOption(string command)
                 .Should()
                 .Pass();
         }
-
-        [Fact]
-        public void ItUsesImplicitRidWhenNoneIsSpecifiedForSelfContained()
```

Comment:
Why are these tests getting deleted?

Maybe we're moving some of the RID inference from the CLI to the SDK, but in that case I would expect to also see some code in the CLI command parsing deleted together with these tests.

Reply:
I deleted the CLI command parsing code when I first added the RID inference but didn't see these tests until now